backend/tests/test_dashboard_legs.py

```
@@ -289,3 +293,177 @@ def test_sorts_itm_before_otm_at_same_dte(self):
+
+    def test_false_when_dte_zero(self):
+        # Zero DTE means the leg expires today — no future window.
+        assert (
+            compute_earnings_in_window(
+                "AAPL", dte=0, earnings_lookup=lambda _t: "2099-01-01"
+            )
+            is False
+        )
+
+    def test_true_when_within_window(self):
+        future = (date.today() + timedelta(days=3)).isoformat()
```

Ruff (0.15.12)

[warning] `datetime.date.today()` used

(DTZ011)